Rivus parser: multi-discriminant discerne not implemented

## Summary

The rivus parser cannot parse multi-discriminant `discerne` statements — syntax where multiple expressions are matched simultaneously with comma-separated patterns. This feature is documented in EBNF.md (lines 204-209) and implemented in faber, but rivus only supports single-discriminant matching. This blocks the bootstrap compiler from compiling `fons/rivus/semantic/typi.fab`.

## Symptom

**Reproduction:**
```bash
bun run build:bootstrap
```

**Error at `semantic/typi.fab:229`:**
```
229:15 - Expected '{': got ','
229:17 - Match cases need 'casu': got 'b'
230:30 - Expected expression: got ','
...
```

**Failing syntax:**
```fab
discerne a, b {
    casu Primitivum ut pa, Primitivum ut pb {
        redde pa.nomen == pb.nomen
    }
    casu Genericum ut ga, Genericum ut gb {
        ...
    }
    casu _, _ {
        redde falsum
    }
}
```

## Root Cause

The rivus parser's `parseDiscerneSententia` in `fons/rivus/parser/sententia/fluxus.fab` does not implement the full grammar:

**Current implementation (lines 132-164):**
```fab
# Parse discriminant expression (SINGLE only)
fixum discriminans = r.expressia()  # <-- should be loop

# Parse variant name (SINGLE only, no comma-separated patterns)
fixum variansNomen = variansNomenSym.valor  # <-- no pattern list
```

**Required grammar (from EBNF.md):**
```ebnf
discerneStmt := 'discerne' discriminants '{' variantCase* defaultCase? '}'
discriminants := expression (',' expression)*    # <-- multiple discriminants
patterns     := pattern (',' pattern)*           # <-- multiple patterns per case
pattern      := '_' | (IDENTIFIER patternBind?)
patternBind  := ('ut' IDENTIFIER) | ('pro' IDENTIFIER (',' IDENTIFIER)*)
```

**Faber's implementation (fons/faber/parser/index.ts:3167-3211):**
```typescript
// Parse comma-separated discriminants
const discriminants: Expression[] = [];
discriminants.push(parseExpression());
while (match('COMMA')) {
    discriminants.push(parseExpression());
}

// Inside case parsing:
const patterns: VariantPattern[] = [];
patterns.push(parseVariantPattern());
while (match('COMMA')) {
    patterns.push(parseVariantPattern());
}
```

## Causal Chain

1. `fons/rivus/semantic/typi.fab` uses multi-discriminant `discerne` for type equality checking (comparing two `SemanticTypus` values)
2. The rivus parser encounters `discerne a, b` and parses `a` as the single discriminant
3. Parser expects `{` but finds `,` → first error
4. Parser continues expecting case content but finds `b` → cascade of errors
5. `typi.fab` fails to parse → its exports are unavailable → semantic errors cascade to all files importing from `./typi`

## Affected Components

**Files using unsupported syntax:**
- `fons/rivus/semantic/typi.fab:229` — `typiAequales()` function

**Components requiring changes:**

| File | Change |
|------|--------|
| `fons/rivus/ast/sententia.fab` | Add `VariansExemplar` type, change `discriminans` to list |
| `fons/rivus/parser/sententia/fluxus.fab` | Implement multi-discriminant and multi-pattern parsing |
| `fons/rivus/codegen/ts/sententia/discerne.fab` | Handle multiple discriminants in codegen |

**Downstream impact:**
- 13 files in `fons/rivus/semantic/` fail with "Module not found: ./typi" due to parse failure
- Bootstrap build blocked (42/107 files fail)

## Fix Considerations

### Approach A: Full multi-discriminant support

Implement the complete EBNF grammar.

**AST changes (`fons/rivus/ast/sententia.fab`):**
```fab
# New pattern type (mirrors faber's VariantPattern)
genus VariansExemplar {
    Locus locus
    textus variansNomen    # variant tag or "_" for wildcard
    bivalens estWildcard
    textus? alias          # 'ut' binding
    lista<textus> vincula  # 'pro' bindings
}

# Updated DiscerneSententia (discriminans becomes list)
DiscerneSententia {
    Locus locus
    lista<Expressia> discriminantes  # was: Expressia discriminans
    lista<VariansCasus> casus
}

# Updated VariansCasus (single fields become pattern list)
genus VariansCasus {
    Locus locus
    lista<VariansExemplar> exemplaria  # was: variansNomen, alias, vincula
    Sententia consequens
}
```

**Parser changes (`fons/rivus/parser/sententia/fluxus.fab`):**
- Parse comma-separated discriminants: `expr (',' expr)*`
- Parse comma-separated patterns per case
- Disambiguate pattern commas from binding commas (look-ahead for `ut`, `pro`, `{`, `ergo`, `reddit`)

**Codegen changes (`fons/rivus/codegen/ts/sententia/discerne.fab`):**
- Generate combined condition: `if (a.tag === 'X' && b.tag === 'Y')`
- Generate bindings for each pattern

**Risk:** Significant restructuring; codegen must handle N discriminants.

### Approach B: Rewrite typi.fab without multi-discriminant

Refactor `typiAequales()` to use nested single-discriminant `discerne` statements.

**Example transformation:**
```fab
# Before (unsupported)
discerne a, b {
    casu Primitivum ut pa, Primitivum ut pb {
        redde pa.nomen == pb.nomen
    }
    casu _, _ { redde falsum }
}

# After (supported)
discerne a {
    casu Primitivum ut pa {
        discerne b {
            casu Primitivum ut pb reddit pa.nomen == pb.nomen
            casu _ reddit falsum
        }
    }
    casu _ reddit falsum
}
```

**Risk:** More verbose, less readable, may miss edge cases. Doesn't fix the parser gap — other code could hit this later.

### Recommendation

Approach A is the correct fix. Multi-discriminant `discerne` is part of the language spec, faber implements it, and it will be needed elsewhere. Approach B is a workaround that accumulates tech debt.

## Open Questions

- Should semantic analysis validate that `patterns.length == discriminants.length`? (faber does this)
- Does the Zig codegen target need parallel updates? (`fons/rivus/codegen/zig/`)

--- a/fons/rivus/ast/sententia.fab
+++ b/fons/rivus/ast/sententia.fab
@@ -196,15 +196,25 @@ genus EligeCasus {
     Sententia consequens         # MassaSententia
 }
 
+# Single pattern in a variant case (matches one discriminant)
+# Example: Primitivum ut p, Click pro x, y, _, Quit
+@ publicum
+genus VariansExemplar {
+    Locus locus
+    textus variansNomen          # variant tag or "_" for wildcard
+    bivalens estWildcard         # true for _ patterns
+    textus? alias                # 'ut' binds entire variant
+    lista<textus> vincula        # field bindings from 'pro'
+}
+
 # Variant case for discerne (pattern matching)
-# Example: si Click pro x, y { ... }
+# Example: casu Click pro x, y { ... }
+# Example: casu Primitivum ut l, Primitivum ut r { ... }  (multi-discriminant)
 @ publicum
 genus VariansCasus {
     Locus locus
-    textus variansNomen
-    textus? alias                # 'ut' binds entire variant
-    lista<textus> vincula        # field bindings from 'pro'
-    Sententia consequens         # MassaSententia
+    lista<VariansExemplar> exemplaria  # patterns (one per discriminant)
+    Sententia consequens               # MassaSententia
 }
 
 # Guard clause
@@ -433,10 +443,11 @@ discretio Sententia {
     }
 
     # Pattern matching statement
-    # Example: discerne event { si Click pro x, y { ... } }
+    # Example: discerne event { casu Click pro x, y { ... } }
+    # Example: discerne a, b { casu X ut x, Y ut y { ... } }  (multi-discriminant)
     DiscerneSententia {
         Locus locus
-        Expressia discriminans
+        lista<Expressia> discriminantes   # one or more expressions to match against
         lista<VariansCasus> casus
     }
 

--- a/fons/rivus/codegen/ts/sententia/discerne.fab
+++ b/fons/rivus/codegen/ts/sententia/discerne.fab
@@ -3,12 +3,23 @@
 # Generates if/else chains for discriminated union matching.
 #
 # TRANSFORMS:
+#   # Single discriminant
 #   discerne x { casu A {...} }       -> if (x.tag === 'A') {...}
 #   discerne x { casu B ut b {...} }  -> if (x.tag === 'B') { const b = x; ... }
 #   discerne x { casu C pro a, b {...} } -> if (x.tag === 'C') { const { a, b } = x; ... }
 #   discerne x { casu _ {...} }       -> else {...}  (wildcard pattern)
+#
+#   # Multi-discriminant
+#   discerne left, right {
+#       casu Primitivum ut l, Primitivum ut r { redde l.nomen == r.nomen }
+#       casu _, _ { redde falsum }
+#   }
+#   -> if (left.tag === 'Primitivum' && right.tag === 'Primitivum') {
+#          const l = left; const r = right;
+#          return l.nomen === r.nomen;
+#      } else { return false; }
 
-ex "../../../ast/sententia" importa Sententia, VariansCasus
+ex "../../../ast/sententia" importa Sententia, VariansCasus, VariansExemplar
 ex "../../../ast/expressia" importa Expressia
 ex "../nucleus" importa TsGenerator
 
@@ -17,48 +28,80 @@ ex "../expressia/index" importa genExpressia
 ex "./index" importa genSententia
 
 @ publica
-functio genDiscerne(Expressia discriminans, lista<VariansCasus> casus, TsGenerator g) -> textus {
-    fixum disc = genExpressia(discriminans, g)
-    varia result = scriptum("§// discerne §\n", g.ind(), disc)
+functio genDiscerne(lista<Expressia> discriminantes, lista<VariansCasus> casus, TsGenerator g) -> textus {
+    # Generate expressions for all discriminants
+    varia discriminantStrs = [] innatum lista<textus>
+    ex discriminantes pro d {
+        discriminantStrs.adde(genExpressia(d, g))
+    }
+
+    varia result = scriptum("§// discerne\n", g.ind())
 
     varia first = verum
     ex casus pro c {
-        # WHY: Wildcard pattern (_) matches anything - generates else block
-        fixum estWildcard = c.variansNomen == "_"
+        # Build combined condition for all patterns
+        varia conditions = [] innatum lista<textus>
+        varia i = 0
+        ex c.exemplaria pro pattern {
+            # Wildcard matches anything - no condition needed
+            si non pattern.estWildcard {
+                fixum disc = discriminantStrs[i]
+                conditions.adde(scriptum("§.tag === '§'", disc, pattern.variansNomen))
+            }
+            i += 1
+        }
 
-        si estWildcard {
-            # Wildcard: generate else block (no condition)
+        # If all patterns are wildcards, this is a catch-all
+        si conditions.longitudo() == 0 {
+            # Generate as else block if not first case, otherwise unconditional
             si first {
-                # If first case is wildcard, just open a block (unconditional)
-                result = scriptum("§§{{\n", result, g.ind())
+                result = scriptum("§§{\n", result, g.ind())
             } secus {
-                result = scriptum("§§else {{\n", result, g.ind())
+                result = scriptum("§§else {\n", result, g.ind())
             }
         } secus {
-            # Normal variant: generate if/else if with tag check
+            fixum condition = conditions.coniunge(" && ")
             si first {
-                result = scriptum("§§if (§.tag === '§') {{\n", result, g.ind(), disc, c.variansNomen)
+                result = scriptum("§§if (§) {\n", result, g.ind(), condition)
                 first = falsum
             } secus {
-                result = scriptum("§§else if (§.tag === '§') {{\n", result, g.ind(), disc, c.variansNomen)
+                result = scriptum("§§else if (§) {\n", result, g.ind(), condition)
             }
         }
 
         g.intraProfundum()
 
-        # Bind alias or fields (only for non-wildcard patterns)
-        si non estWildcard {
-            si nonnihil c.alias {
-                # WHY: 'ut' binds the whole variant for access via alias.
-                result = scriptum("§§const § = §;\n", result, g.ind(), c.alias, disc)
-            } sin c.vincula.longitudo() > 0 {
-                result = scriptum("§§const {{ § }} = §;\n", result, g.ind(), c.vincula.coniunge(", "), disc)
+        # Generate bindings for each pattern
+        varia j = 0
+        ex c.exemplaria pro pattern {
+            # Skip wildcards - no bindings
+            si non pattern.estWildcard {
+                fixum disc = discriminantStrs[j]
+                result = scriptum("§§", result, genPatternBindings(pattern, disc, g))
             }
+            j += 1
         }
 
+        # Generate body statements
         result = scriptum("§§\n", result, genSententia(c.consequens, g))
         g.exiProfundum()
-        result = scriptum("§§}}\n", result, g.ind())
+        result = scriptum("§§}\n", result, g.ind())
+    }
+
+    redde result
+}
+
+# Generate bindings for a single pattern
+functio genPatternBindings(VariansExemplar pattern, textus discriminant, TsGenerator g) -> textus {
+    varia result = ""
+
+    # Alias binding: casu Click ut c
+    si nonnihil pattern.alias {
+        result = scriptum("§const § = §;\n", g.ind(), pattern.alias, discriminant)
+    }
+    # Positional bindings: casu Click pro a, b
+    sin pattern.vincula.longitudo() > 0 {
+        result = scriptum("§const { § } = §;\n", g.ind(), pattern.vincula.coniunge(", "), discriminant)
     }
 
     redde result

--- a/fons/rivus/codegen/ts/sententia/index.fab
+++ b/fons/rivus/codegen/ts/sententia/index.fab
@@ -130,7 +130,7 @@ functio genSententiaContent(Sententia stmt, TsGenerator g) -> textus {
         }
 
         casu DiscerneSententia ut s {
-            redde genDiscerne(s.discriminans, s.casus qua lista<VariansCasus>, g)
+            redde genDiscerne(s.discriminantes, s.casus qua lista<VariansCasus>, g)
         }
 
         casu CustodiSententia ut s {

--- a/fons/rivus/parser/sententia/fluxus.fab
+++ b/fons/rivus/parser/sententia/fluxus.fab
@@ -16,9 +16,10 @@
 # - custodi = guard (imperative of custodire)
 
 ex "../resolvitor" importa Resolvitor
+ex "../nucleus" importa Parser
 ex "../../ast/positio" importa Locus
 ex "../../ast/expressia" importa Expressia
-ex "../../ast/sententia" importa Sententia, EligeCasus, VariansCasus, CustodiClausula, CapeClausula
+ex "../../ast/sententia" importa Sententia, EligeCasus, VariansCasus, VariansExemplar, CustodiClausula, CapeClausula
 ex "../../ast/lexema" importa SymbolumGenus
 ex "../errores" importa ParserErrorCodice
 ex "./massa" importa parseMassaSententia
@@ -112,14 +113,25 @@ functio parseEligeSententia(Resolvitor r) -> Sententia {
 # Parse discerne (pattern match) statement
 #
 # GRAMMAR:
-#   discerneStmt := 'discerne' expr '{' variantCase* '}'
-#   variantCase := 'casu' IDENT ('pro' IDENT (',' IDENT)*)? block
+#   discerneStmt := 'discerne' discriminants '{' variantCase* '}'
+#   discriminants := expression (',' expression)*
+#   variantCase := 'casu' patterns (block | 'ergo' stmt | 'reddit' expr)
+#   patterns := pattern (',' pattern)*
+#   pattern := '_' | (IDENTIFIER patternBind?)
+#   patternBind := ('ut' IDENTIFIER) | ('pro' IDENTIFIER (',' IDENTIFIER)*)
 #
 # Examples:
+#   # Single discriminant
 #   discerne event {
 #       casu Click pro x, y { scribe scriptum("clicked at §, §", x, y) }
 #       casu Keypress ut key { scribe scriptum("pressed §", key) }
 #       casu Quit { mori "goodbye" }
+#   }
+#
+#   # Multi-discriminant
+#   discerne left, right {
+#       casu Primitivum ut l, Primitivum ut r { redde l.nomen == r.nomen }
+#       casu _, _ { redde falsum }
 #   }
 @ publica
 functio parseDiscerneSententia(Resolvitor r) -> Sententia {
@@ -129,8 +141,12 @@ functio parseDiscerneSententia(Resolvitor r) -> Sententia {
     # Consume 'discerne'
     p.expectaVerbum("discerne", ParserErrorCodice.ExpectaturDiscerne)
 
-    # Parse discriminant expression
-    fixum discriminans = r.expressia()
+    # Parse comma-separated discriminants
+    varia discriminantes = [] innatum lista<Expressia>
+    discriminantes.adde(r.expressia())
+    dum p.congruet(SymbolumGenus.Coma) {
+        discriminantes.adde(r.expressia())
+    }
 
     # Expect opening brace
     p.expecta(SymbolumGenus.UncusSin, ParserErrorCodice.ExpectaturUncusSin)
@@ -143,24 +159,11 @@ functio parseDiscerneSententia(Resolvitor r) -> Sententia {
             fixum casusLocus = p.locusActualis()
             p.procede()  # Consume 'casu'
 
-            # Parse variant name
-            fixum variansNomenSym = p.expectaNomenAutVerbum(ParserErrorCodice.ExpectaturNomen)
-            fixum variansNomen = variansNomenSym.valor
-
-            # Parse optional bindings: ut alias OR pro x, y, z
-            # WHY: 'ut' binds the whole variant, 'pro' binds fields positionally.
-            varia alias = nihil qua textus?
-            varia vincula = [] innatum lista<textus>
-            si p.probaVerbum("ut") {
-                p.procede()
-                fixum aliasSym = p.expectaNomenAutVerbum(ParserErrorCodice.ExpectaturNomen)
-                alias = aliasSym.valor
-            } sin p.probaVerbum("pro") {
-                p.procede()
-                fac {
-                    fixum vincSym = p.expectaNomenAutVerbum(ParserErrorCodice.ExpectaturNomen)
-                    vincula.adde(vincSym.valor)
-                } dum p.congruet(SymbolumGenus.Coma)
+            # Parse comma-separated patterns (one per discriminant)
+            varia exemplaria = [] innatum lista<VariansExemplar>
+            exemplaria.adde(parseVariansExemplar(p))
+            dum p.congruet(SymbolumGenus.Coma) {
+                exemplaria.adde(parseVariansExemplar(p))
             }
 
             # Parse body
@@ -168,9 +171,7 @@ functio parseDiscerneSententia(Resolvitor r) -> Sententia {
 
             casus.adde({
                 locus: casusLocus,
-                variansNomen: variansNomen,
-                alias: alias,
-                vincula: vincula,
+                exemplaria: exemplaria,
                 consequens: consequens
             } qua VariansCasus)
         } secus {
@@ -183,9 +184,107 @@ functio parseDiscerneSententia(Resolvitor r) -> Sententia {
 
     redde finge DiscerneSententia {
         locus: locus,
-        discriminans: discriminans,
+        discriminantes: discriminantes,
         casus: casus
     } qua Sententia
+}
+
+# Parse a single variant pattern
+#
+# GRAMMAR:
+#   pattern := '_' | (IDENTIFIER patternBind?)
+#   patternBind := ('ut' IDENTIFIER) | ('pro' IDENTIFIER (',' IDENTIFIER)*)
+#
+# DISAMBIGUATION: After 'pro', commas separate bindings until we see:
+#   - '_' (wildcard pattern)
+#   - An identifier followed by 'ut' or 'pro' (new pattern with binding)
+#   - '{', 'ergo', 'reddit' (end of patterns)
+functio parseVariansExemplar(Parser p) -> VariansExemplar {
+    fixum locus = p.locusActualis()
+
+    # Parse variant name (or wildcard)
+    fixum variansNomenSym = p.expectaNomenAutVerbum(ParserErrorCodice.ExpectaturNomen)
+    fixum variansNomen = variansNomenSym.valor
+
+    # Check for wildcard
+    si variansNomen == "_" {
+        redde {
+            locus: locus,
+            variansNomen: "_",
+            estWildcard: verum,
+            alias: nihil qua textus?,
+            vincula: [] innatum lista<textus>
+        } qua VariansExemplar
+    }
+
+    # Parse optional bindings: ut alias OR pro x, y, z
+    # WHY: 'ut' binds the whole variant, 'pro' binds fields positionally.
+    varia alias = nihil qua textus?
+    varia vincula = [] innatum lista<textus>
+
+    si p.probaVerbum("ut") {
+        p.procede()
+        fixum aliasSym = p.expectaNomenAutVerbum(ParserErrorCodice.ExpectaturNomen)
+        alias = aliasSym.valor
+    } sin p.probaVerbum("pro") {
+        p.procede()
+        fac {
+            fixum vincSym = p.expectaNomenAutVerbum(ParserErrorCodice.ExpectaturNomen)
+            vincula.adde(vincSym.valor)
+        } dum estProximumVinculum(p)
+    }
+
+    redde {
+        locus: locus,
+        variansNomen: variansNomen,
+        estWildcard: falsum,
+        alias: alias,
+        vincula: vincula
+    } qua VariansExemplar
+}
+
+# Check if the token after a comma is a binding (not a new pattern)
+#
+# WHY: In multi-discriminant patterns, commas separate both:
+#   - Bindings within 'pro': `casu Click pro x, y { ... }`
+#   - Patterns: `casu Click ut c, Quit { ... }`
+#
+# RULE: After comma, it's a new pattern ONLY if:
+#   - Next token is '_' (wildcard)
+#   - Next token is identifier followed by 'ut' or 'pro' (pattern with binding)
+# Otherwise it's another binding.
+functio estProximumVinculum(Parser p) -> bivalens {
+    # Must be at comma to continue
+    si non p.proba(SymbolumGenus.Coma) {
+        redde falsum
+    }
+
+    fixum proximum = p.specta(1)
+
+    # Comma followed by '_' is next pattern (wildcard)
+    si proximum.species == SymbolumGenus.Nomen et proximum.valor == "_" {
+        redde falsum
+    }
+
+    # Comma followed by non-identifier is not a binding
+    si proximum.species != SymbolumGenus.Nomen et proximum.species != SymbolumGenus.Verbum {
+        redde falsum
+    }
+
+    # Look at what follows the identifier
+    fixum postIdent = p.specta(2)
+
+    # If followed by 'ut' or 'pro', it's a new pattern with binding
+    si postIdent.species == SymbolumGenus.Verbum {
+        si postIdent.valor == "ut" aut postIdent.valor == "pro" {
+            redde falsum
+        }
+    }
+
+    # Otherwise assume it's a binding (including when followed by ',', '{', 'ergo', 'reddit')
+    # Consume the comma and return true to continue the binding loop
+    p.procede()
+    redde verum
 }
 
 # ============================================================================

--- a/fons/rivus/semantic/sententia/imperium.fab
+++ b/fons/rivus/semantic/sententia/imperium.fab
@@ -179,34 +179,45 @@ functio analyzeDiscerne(Resolvitor r, Sententia discerneStmt) -> vacuum {
 
     discerne discerneStmt {
         casu DiscerneSententia ut d {
-            # Analyze discriminant
-            fixum discTypus = r.expressia(d.discriminans)
+            # Analyze all discriminants
+            ex d.discriminantes pro disc {
+                r.expressia(disc)
+            }
 
             # Analyze each variant case
             ex d.casus pro casus {
                 # Enter scope for pattern bindings
                 a.intraScopum(ScopusSpecies.Massa)
 
-                # Define bound variables if present
-                # WHY: 'ut' introduces an alias, 'pro' introduces field bindings.
-                si nonnihil casus.alias {
-                    fixum alias = casus.alias qua textus
-                    a.scopus.symbola[alias] = {
-                        nomen: alias,
-                        semanticTypus: IGNOTUM,  # TODO: infer from variant
-                        species: SymbolumSpecies.Variabilis,
-                        mutabilis: falsum,
-                        locus: casus.locus
+                # Define bound variables for all patterns in this case
+                # WHY: Each pattern can introduce 'ut' alias or 'pro' field bindings.
+                ex casus.exemplaria pro pattern {
+                    # Skip wildcards - they have no bindings
+                    si pattern.estWildcard {
+                        perge
                     }
-                }
 
-                ex casus.vincula pro vinculum {
-                    a.scopus.symbola[vinculum] = {
-                        nomen: vinculum,
-                        semanticTypus: IGNOTUM,  # TODO: infer from variant
-                        species: SymbolumSpecies.Variabilis,
-                        mutabilis: falsum,
-                        locus: casus.locus
+                    # Define alias if present
+                    si nonnihil pattern.alias {
+                        fixum alias = pattern.alias qua textus
+                        a.scopus.symbola[alias] = {
+                            nomen: alias,
+                            semanticTypus: IGNOTUM,  # TODO: infer from variant
+                            species: SymbolumSpecies.Variabilis,
+                            mutabilis: falsum,
+                            locus: pattern.locus
+                        }
+                    }
+
+                    # Define field bindings if present
+                    ex pattern.vincula pro vinculum {
+                        a.scopus.symbola[vinculum] = {
+                            nomen: vinculum,
+                            semanticTypus: IGNOTUM,  # TODO: infer from variant
+                            species: SymbolumSpecies.Variabilis,
+                            mutabilis: falsum,
+                            locus: pattern.locus
+                        }
                     }
                 }
 

